nautobot/ipam/views.py

```diff
@@ -1277,7 +1277,7 @@ class VLANGroupBulkDeleteView(generic.BulkDeleteView):
 
 
 class VLANListView(generic.ObjectListView):
-    queryset = VLAN.objects.select_related("location", "vlan_group", "tenant", "role", "status")
+    queryset = VLAN.objects.select_related("vlan_group", "tenant", "role", "status")
```

Comment:
Probably need a `count_related` here and below once you add the `locations` `LinkedCountColumn` to the table.